docs/ddp/README.md

```diff
@@ -1,90 +1,76 @@
+# SR-IOV Network Device plugin with DDP
+Dynamic Device Personalizationi aka DDP allows dynamic reconfiguration of the packet processing pipeline of Intel® Ethernet 800/700 series to meet specific use-case needs for on-demand, adding new packet processing pipeline configuration *packages* to a network adapter at run time, without resetting or rebooting the server.
 
+The SR-IOV Network Device plugin can be used to identify currently running DDP *packages*, allowing it to filter Virtual Functions (VFs) by their DDP package name.
 
-# SR-IOV network device plugin with DDP
-Dynamic Device Personalization aka DDP allows dynamic reconfiguration of the packet processing pipeline of Intel Ethernet 700 Series to meet specific use case needs on demand, adding new packet processing pipeline configuration *profiles* to a network adapter at run time, without resetting or rebooting the server.
+In this documentation we will cover the kernel driver use-case only. DPDK configuration of DDP is out of scope.
 
-(ref: [Dynamic Device Personalization for Intel® Ethernet 700 Series](https://software.intel.com/en-us/articles/dynamic-device-personalization-for-intel-ethernet-700-series))
+## Dynamic Device Personalization for Intel® Ethernet Controller E810
+For Intel® Ethernet Controller E810, a DDP package can be loaded into the NIC using the ice kernel driver. Current device DDP state can be determined with DDPTool or devlink.
```

Comment:
Do we want to add devlink support in sriov device plugin?

Reply:
@zshi-redhat Yes. It is currently being worked on by Patryk. I discussed it at the last community meeting.